tests/unit/Makefile.am

```diff
@@ -10,17 +10,17 @@ test_noise_SOURCES = \
 	test-main.c \
 	test-names.c \
 	test-patterns.c \
-	test-protobufs.c \
 	test-randstate.c \
 	test-signstate.c \
 	test-symmetricstate.c
+# test-protobufs.c 
 
 AM_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/src
 AM_CFLAGS = @WARNING_FLAGS@
 
 LDADD = ../../src/keys/libnoisekeys.a \
-        ../../src/protobufs/libnoiseprotobufs.a \
         ../../src/protocol/libnoiseprotocol.a
+# ../../src/protobufs/libnoiseprotobufs.a 
```

Comment:
This line has been commented. I believe that in our case, it should be explained why this tests have been commented. Moreover, the `test-protobufs.c` and `../../src/protobufs/libnoiseprotobufs.a` should be generated if and only if the library is compiled for scripts or the vault app. This is important for backward compatibility.